test/testAll.m

```diff
@@ -22,12 +21,13 @@
     CBTDIR = pth(1:end-(length('testAll.m') + 1));
 
     % change the directory to the root
-    cd([CBTDIR, filesep, '..', filesep]);
-
-    % include the root folder and all subfolders
-    addpath(genpath(pwd));
+    cd([CBTDIR, filesep, '..', filesep]);    
+else
+    CBTDIR = fileparts(which('initCobraToolbox.m'));
+    cd(CBTDIR);
 end
-
+%include the root folder and all subfolders.
+addpath(genpath(pwd));
 CBTDIR = fileparts(which('initCobraToolbox.m'));
```

Comment:
I am not sure that it is a good idea to redefine `CBTDIR` with an addition of `/test`. Whenever `/test` is needed, could we not have `[CBTDIR filesep 'test']`? 

Simplifies maintenance when `CBTDIR` always points to the root.